worker/fbs/plainTransport.fbs

```diff
@@ -46,6 +46,17 @@ table GetStatsResponse {
 
 // Notifications from Worker.
 
+union Notification {
+   Tuple: TupleNotification,
+   RtcpTuple: RtcpTupleNotification,
+   SctpStateChange: FBS.Transport.SctpStateChangeNotification,
+   Trace: FBS.Transport.TraceNotification,
+}
+
+table NotificationWrapper {
```

Comment:
This wrapper is needed since `Notification`, which is a **union** cannot be used by itself, it needs to be contained within a *table**.